tests/core/test_model.py

```diff
@@ -99,7 +99,7 @@ def test_load(assert_exp_eq):
     assert model.owner == "owner_name"
     assert model.dialect == "spark"
     assert model.storage_format == "iceberg"
-    assert [col.sql() for col in model.partitioned_by] == ["a", "d"]
+    assert [col.sql() for col in model.partitioned_by] == ["a", '"d"']
```

Comment:
why is only d quoted? is it because a is inferred? this again seems problematic?

Reply:
`a` is a time column which is a plan string in the model definition.